scripts/download-redirects.ts

```diff
@@ -37,10 +45,41 @@ async function fetchRedirectRules() {
   return redirectRules;
 }
 
+async function fetchFileRedirectRules() {
+  const redirectRules: RedirectRule[] = [];
+    
+  for await (const item of client.items.listPagedIterator({ filter: { type: 'file' } }) as unknown as FileRecord[]) {
+    if (!item.slug) {
+      continue;
+    }
+    const file = await client.uploads.find(item.file.upload_id);
+
+    redirectRules.push({
+      from: item.slug,
+      to: file.url,
+      statusCode: '200',
+    });
+  }
+  return redirectRules;
+}
+
 async function downloadRedirectRules() {
-  const redirectRules = await fetchRedirectRules();
-  const cloudflareRedirectFile = redirectRules.map(rule => `${rule.from} ${rule.to} ${rule.statusCode}`).join('\n');
-  await writeFile('./dist/_redirects', cloudflareRedirectFile);
+  const fileRedirects = await fetchFileRedirectRules();
+  const redirects = fetchRedirectRules();
+  const existingFile = await readFile('./dist/_redirects', 'utf-8');
+
+  const combinedFile = [
```

Comment:
Well, this is disappointing. Apparently Cloudflare only supports pointing to relative URLs (so same-domain) 🤷 :

```
19:55:58.744 | Found invalid redirect lines:
-- | --
19:55:58.744 | - #2: /files/* https://www.datocms-assets.com/:splat 200
19:55:58.745 | Proxy (200) redirects can only point to relative paths. Got https://www.datocms-assets.com/:splat
19:55:58.745 | - #6: /my-files/example.pdf https://www.datocms-assets.com/104869/1718125906-dummy-with-custom-slug.pdf 200
19:55:58.745 | Proxy (200) redirects can only point to relative paths. Got https://www.datocms-assets.com/104869/1718125906-dummy-with-custom-slug.pdf
``` 

Full log: https://dash.cloudflare.com/e20ebc96b0121ea21bf59ea0a95e7997/pages/view/head-start/c7222c50-8f41-4325-b2d9-e5e019854e7c

Reply:
[Here it looks as if everything is fine](https://dash.cloudflare.com/e20ebc96b0121ea21bf59ea0a95e7997/pages/view/head-start/c7222c50-8f41-4325-b2d9-e5e019854e7c/redirects) :-/

![image](https://github.com/voorhoede/head-start/assets/1516059/357666cf-5598-47ee-8434-2c8f38b9830b)

Reply:
Ow, [here it is](https://developers.cloudflare.com/pages/configuration/redirects/) :)

> **Proxying**
> Proxying will only support relative URLs on your site. You cannot proxy external domains.

[Less powerful than Netlify](https://docs.netlify.com/routing/redirects/rewrites-proxies/).